kolibri/deployment/default/settings/base.py

```diff
@@ -191,6 +191,6 @@
     }
 }
 
-AUTH_USER_MODEL = 'kolibriauth.BaseUser'
+AUTH_USER_MODEL = 'kolibriauth.DeviceOwner'
```

Comment:
The choice for `AUTH_USER_MODEL` was fairly arbitrary, but since I'm guessing the main place it would come into play for us is in the Django admin, and `DeviceOwner` is more likely to be granted access to the Django admin than a `FacilityUser`, I might as well put that.